index.html

```diff
@@ -27,42 +78,63 @@ <h1 class="hero__title">planet</h1>
                     <div class="hero__actions"> 
                         <a href="#" class="hero__button hero__button--green">Find out more</a>            
                         <a href="#" class="hero__button hero__button--white">Play video</a>  
-                    </div>            
-                   
+                    </div>     
+                    <span ><a  class="hero__down--arrow" href="#gallery"> &#8595;</a></span>        
                 </div>
-                <div class="sidebar"> 
-                    <ul class="style">
-                        <li  class="sidebar__text "> Wild Avenue, San Francisco, CA</li>                    
-                        <li  class="sidebar__text "> wild@life.com</li>
-                        <li  class="sidebar__text "> +40 070 991 598</li>
-                        
+                <div class="hero__sidebar"> 
+                    <ul class="hero__sidebar--apparence">
+                        <li  class="hero__sidebar--text"> Wild Avenue, San Francisco, CA</li>                    
+                        <li  class="hero__sidebar--text"> wild@life.com</li>
+                        <li  class="hero__sidebar--text"> +40 070 991 598</li>                   
                     </ul>    
-                   
                 </div>
-            </div>      
+            </div>     
         </section> 
         <section class="mission">
-            <h1 class="mission__title">Our Mission</h1>
-            <div class="mission__content">
-                <p class="mission__text">
+                <h1 class="mission__title">Our Mission</h1>
+                <div class="mission__content">
+                    <p class="mission__text">
                     Our wildlife is important to us
                     if we don’t take care of it
                     maybe one day will turn
                     against us and we will die.
-                </p>
-                <p class="mission__text">
+                    </p>
+                     <p class="mission__text">
                     Our wildlife is important to us
                     if we don’t take care of it
                     maybe one day will turn
                     against us and we will die.
-                </p>
-                <p class="mission__text">
+                    </p>
+                    <p class="mission__text">
                     Our wildlife is important to us
                     if we don’t take care of it
                     maybe one day will turn
                     against us and we will die.
-                </p>
+                    </p>
+            </div>
+        </section>
+        <section id="gallery" class="gallery">
```

Comment:
no need of an ID

Reply:
aici am folosit id-ul pentru sageata care duce in josul paginii, catre galerie